### ISSUE TITLE
Output option does not work

### ISSUE THREAD

**Issue by Akrion** [smartprocure/directory-metagen#16]
```
{
    path: __dirname + '/public/react-components/babel-src/reports/apps',
    output: '__reportCatalog.js',
    format: es6Format,
    exclusions: [
        '__reportCatalog.js'
    ]
}
```

In the above scenario the output ends up being `apps__reportCatalog.js` file located in the reports folder. The desired result is a file named `__reportCatalog.js` located in the apps folder.

**Comment by daedalus28** [comment]
We should definitely fix this.

**Comment by Akrion** [comment]
Fixed with PR #23
